src/templates/toolkit/bet/static.html.tt

```diff
@@ -115,7 +115,7 @@
                            </label>
                         </div>
                         <div class="big-col">
-                           <input id="barrier_low" type="text" name="L" pattern="^\-\d+(\.\d{1,3})?|\d+(\.\d{1,3})?$" title="0.001 format" />
+                           <input id="barrier_low" type="text" name="L" pattern="^\-\d+(\.\d{1,4})?|\d+(\.\d{1,4})?$" title="maximum of 4 decimal points allowed" />
```

Comment:
if it means always starts with `-` then it may fail because sometimes we show absolute value as well